docs/docs-components/COMPONENT_DATA.js

```diff
@@ -4032,7 +4032,7 @@ const UTILITIES_LIST: $ReadOnlyArray<ListItemType> = [
     svg: <ProviderHandlers />,
     name: 'GlobalEventsHandlerProvider',
     aliases: [],
-    previouslyNamed: [],
+    previouslyNamed: ['OnLinkNavigationProvider'],
```

Comment:
This is helpful, but unrelated to the PR as a whole. We want to keep PRs focused and small, which makes it easier to see what changes are being made, and to revert those changes in the future if needed.

You can leave this change, but in the future please put unrelated cleanup in a separate PR (but thank you for the cleanup!!).